src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+#spring.datasource.url=jdbc:h2:file:/data/demo
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
```

Comment:
to see your in-memory db, go to localhost:8080/h2-console when the app is running. Fill out the info in this application properties file (more than just username and password) to access it. default username is 'sa' and default password is: '' (no password; leave it blank).
You can store the db to a file by uncommenting the second line and commenting out the first. Be sure to swap it back once you're done unless you actually want to change it for everyone.